### ISSUE TITLE
Can't shout non-english sentences if they begin with Cyrillic character

### ISSUE THREAD

**Issue by KanediasTheMaker** [topkecleon/otouto#66]
E.g.

```
/shout ЖИЗНЬ - not working
/shout TEST ЖИЗНЬ - works
```

**Comment by TiagoDanin** [comment]
![image](https://cloud.githubusercontent.com/assets/5731176/15654042/8a455378-2656-11e6-86ef-7e4a6ff35b9c.png)
Worked!

**Comment by KanediasTheMaker** [comment]
Strange, try this please:
```
/shout ЖИЗНЬ ВОРАМ
```

**Comment by bb010g** [comment]
Verified broken.
![It's borked](https://cloud.githubusercontent.com/assets/340132/15787566/f615c262-2977-11e6-95b6-4adf1293ccf0.PNG)

**Comment by topkecleon** [comment]
Seems to be fixed by #68. Advise me of any future problems.
